tests/ignite/handlers/test_checkpoint.py

```diff
@@ -1211,6 +1211,158 @@ def test_distrib_single_device_xla_nprocs(xmp_executor, dirname):
     xmp_executor(_test_tpu_saves_to_cpu_nprocs, args=(dirname,), nprocs=n)
 
 
+def test_checkpoint_filename_pattern():
```

Comment:
It would be necessary to add tests for `setup_filename_pattern` as it is a public methods.